test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -949,8 +949,9 @@ protected void assertLastSchemaChange(Event.SchemaChange.Change change, Event.Sc
                                           String keyspace, String name,
                                           String... argTypes)
     {
-        Assert.assertTrue(lastSchemaChangeResult instanceof ResultMessage.SchemaChange);
-        ResultMessage.SchemaChange schemaChange = (ResultMessage.SchemaChange) lastSchemaChangeResult;
+        ResultMessage schemaChangeRes = recordedSchemaChangeResult;
+        Assert.assertTrue(schemaChangeRes instanceof ResultMessage.SchemaChange);
+        ResultMessage.SchemaChange schemaChange = (ResultMessage.SchemaChange) schemaChangeRes;
```

Comment:
If we use `Event.SchemaChange schemaChange = ((ResultMessage.SchemaChange) recordedSchemaChangeResult).change;` we can simplify the following code by removing the `.change` parts.